docs/assets/docs.scss

```diff
@@ -0,0 +1,57 @@
+body {
+    margin: 0;
```

Comment:
Make sure that Sass Lint can get to this file (`.sass-lint.yml`) and please fix any linting issues that come up